### ISSUE TITLE
Task II.A中的数据集

### ISSUE THREAD

**Issue by Sh3lock** [JunlinHan/BID#11]
我没理解错的话，这个训练数据是随机的，每一次都不一样，这是否会对每次的训练效果产生影响呢？

**Comment by JunlinHan** [comment]
Yes, it is generated on-the-fly. When it trains for longer epochs (and sees lots of samples) the performance should be similar.
